src/adapters/geocoder.js

```diff
@@ -82,10 +82,11 @@ export function getGeocoderSuggestions(term, { focus = {}, useNlu = false } = {}
           return new BragiPoi(feature, queryContext);
         });
         const bragiResponse = { pois };
-        if (intentions) {
-          bragiResponse.intentions = intentions
-            .map(intention => new Intention(intention))
-            .filter(intention => intention.isValid());
+        if (intention) {
+          const parsed = new Intention(intention);
```

Comment:
BTW I tried to write `const intention = new Intention(intention);`, while I wasn't expecting Javascript to support shadowing, I'm quite surprised it just made the function abort silently, without a warning or crashing the application. Is this the expected behavior, even though I built using the `--mode=development` parameter?